src/cunumeric/mapper.cc

```diff
@@ -96,6 +105,23 @@ std::vector<StoreMapping> CuNumericMapper::store_mappings(
       } else
         return {};
     }
+    case CUNUMERIC_MATMUL:
+    case CUNUMERIC_MATVECMUL: {
+      // TODO: Our actual requirements are a little less strict than this; we require each array or
+      // vector to have a stride of 1 on at least one dimension.
+      std::vector<StoreMapping> mappings;
+      auto& inputs  = task.inputs();
+      auto& outputs = task.outputs();
+      for (auto& input : inputs) {
+        mappings.push_back(StoreMapping::default_mapping(input, options.front()));
+        mappings.back().policy.exact = true;
+      }
+      for (auto& output : outputs) {
```

Comment:
A minor technical detail: the output would already be mapped to an exact instance by the previous fill task and there would be no other tasks in between that would bloat it. So, technically this policy isn't necessary, or if we're gonna enforce it, we should also apply it to the preceding fill task. That said, I'm not suggesting we do that, since we don't support context sensitive mapping decisions quite yet.